python/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(XCFUN_SWIG_WRAP "xcfun_swigPYTHON_wrap.c")
 add_custom_command(
   OUTPUT ${XCFUN_SWIG_WRAP}
   COMMAND "${SWIG_EXECUTABLE}"
-  ARGS "-python" "-I${PROJECT_SOURCE_DIR}/include" "-outdir" "${CMAKE_CURRENT_BINARY_DIR}" "-o" ${XCFUN_SWIG_WRAP} ${XCFUN_SWIG}
+  ARGS "-python" "-I${CMAKE_BINARY_DIR}/include" "-I${PROJECT_SOURCE_DIR}/api" "-outdir" "${CMAKE_CURRENT_BINARY_DIR}" "-o" ${XCFUN_SWIG_WRAP} ${XCFUN_SWIG}
```

Comment:
Apparently SWIG takes over completely the build of this wrapper DSO. Hence I cannot fix the `RPATH` using CMake (which would be one line of CMake). Plus, SWIG uses GCC no matter what compiler was used to build XCFun. The latter is not a problem (I think?) but is certainly worrisome. Until we explore pybind11 in a little more detail this will have to do.

Reply:
Again, now CMake is in charge, but I am not sure whether this is a good or bad thing. In this step, one has to link against the Python libraries these are usually built with gcc. So switching to another compiler might cause problems here, but didnt test other compilers.